### ISSUE TITLE
HTTP 400 Bad Request when trying to revoke credential from multisig AID

### ISSUE THREAD

**Issue by lenkan** [WebOfTrust/keria#146]
# Steps to reproduce:

1. Create multisig AID "issuer"
2. Create single sig aid "holder"
3. Issue credential from "issuer"
4. Revoke credential from "issuer"

# Actual result

Empty error in signify:

```
  Error
      at SignifyClient.fetch (/home/lenkan/code/keri-sandbox/node_modules/signify-ts/src/keri/app/clienting.ts:213:19)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

No error logging in keria.

## What actually happens.

Signify receives a HTTP 400 error (this is currently not visible in the stack trace see https://github.com/WebOfTrust/signify-ts/issues/181). The HTTP 400 is generated by keria from here: https://github.com/WebOfTrust/keria/blob/development/src/keria/app/credentialing.py#L675. After adding logging, the error is `'dict' object has no attribute 'sn'`. Which originates from here: https://github.com/WebOfTrust/keria/blob/development/src/keria/app/credentialing.py#L811

# Expected result

I think there are multiple things that needs to be addressed:

- HTTP information in error from client: https://github.com/WebOfTrust/signify-ts/issues/181
- Caught exception should not generate a HTTP 400 unless it is a client input error. I think this looks like an HTTP 400 Internal Server Error. Correct me if I am wrong.
- The root cause, I can see the `anc` dict does not have an `sn` property, only an `s` property. Not sure if the fix is just to replace it or if there is a deeper issue.

**Comment by lenkan** [comment]
I have created a reproduction script in signify here: https://github.com/WebOfTrust/signify-ts/pull/202

**Comment by pfeairheller** [comment]
Fixed with PR #197
